core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -39,6 +40,11 @@ class EMConfig {
 
         private const val headerRegex = "(.+:.+)|(^$)"
 
+        private const val targetSeparator = ";"
+        private const val targetNone = "\\b(None|NONE|none)\\b"
+        private const val targetPrefix = "\\b(Class|CLASS|class|Line|LINE|line|Branch|BRANCH|branch|MethodReplacement|METHODREPLACEMENT|method[r|R]eplacement|Success_Call|SUCCESS_CALL|success_[c|C]all|Local|LOCAL|local|PotentialFault|POTENTIALFAULT|potential[f|F]ault)\\b"
```

Comment:
hmmm, not sure about this one. if modify the names in `ObjectiveNaming`, this will break without any warning, isn't it?
also, to ignore case in a Java regex, can easily use `(?i)`, see [https://www.baeldung.com/java-case-insensitive-string-matching]()

Reply:
does eg, `cLAss` satisfy regex?

the test `testValidTargetExclusions` is used to check if there is any update existing in the `ObjectiveNaming`